src/main/java/seedu/address/model/Model.java

```diff
@@ -90,4 +90,6 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    void recordCommand(String userInput);
```

Comment:
I think it would be good to include JavaDocs.

Reply:
Fixed.